integration_tests/requirements.txt

```diff
@@ -1,7 +1,7 @@
-dbt-snowflake>=1.3.0,<2.0.0
-dbt-bigquery>=1.3.0,<2.0.0
-dbt-redshift>=1.3.0,<2.0.0
-dbt-postgres>=1.3.0,<2.0.0
-dbt-spark>=1.3.0,<2.0.0
-dbt-spark[PyHive]>=1.3.0,<2.0.0
-dbt-databricks>=1.6.0,<2.0.0
\ No newline at end of file
+dbt-snowflake>=1.3.0,<1.8.0
+dbt-bigquery>=1.3.0,<1.8.0
+dbt-redshift>=1.3.0,<1.8.0
+dbt-postgres>=1.3.0,<1.8.0
+dbt-spark>=1.3.0,<1.8.0
+dbt-spark[PyHive]>=1.3.0,<1.8.0
+dbt-databricks>=1.6.0,<1.8.0
```

Comment:
Limiting this for the time being. Until we are able to figure out what changes we need to make for 1.8.0 compatibility in our integration tests